pkg/addon/configpolicy/manifests/managedclusterchart/templates/deployment.yaml

```diff
@@ -58,8 +61,33 @@ spec:
           - name: NO_PROXY
             value: {{ .Values.global.proxyConfig.NO_PROXY }}
           {{- end }}
-        livenessProbe: null
-        readinessProbe: null
+        livenessProbe:
+          httpGet:
+            path: /healthz
+            port: 8081
+          failureThreshold: 3
+          periodSeconds: 10
+          {{- if semverCompare "< 1.20.0" .Capabilities.KubeVersion.Version }}
+          initialDelaySeconds: 300
+          {{- end }}
+        readinessProbe:
+          httpGet:
+            path: /readyz
+            port: 8081
+          failureThreshold: 3
+          periodSeconds: 10
+          {{- if semverCompare "< 1.20.0" .Capabilities.KubeVersion.Version }}
+          initialDelaySeconds: 300
+          {{- end }}
+        {{- if semverCompare ">= 1.20.0" .Capabilities.KubeVersion.Version }}
+        {{- /* startupProbe became stable in k8s 1.20 */}}
+        startupProbe:
+          httpGet:
+            path: /healthz
```

Comment:
Should this be `/readyz`?

Reply:
Sometimes they're exactly the same: https://github.com/open-cluster-management-io/config-policy-controller/blob/main/main.go#L222-L230

Sometimes they're just slightly different: https://github.com/open-cluster-management-io/governance-policy-spec-sync/blob/main/main.go#L233-L242

But basically I think either could work. If you have a strong preference, I could change it.

Reply:
The reason I ask is that startup is equivalent to ready in my opinion.